addons/crm/models/crm_team.py

```diff
@@ -613,6 +613,8 @@ def _assign_and_convert_leads(self, force_quota=False):
             ['id:array_agg'],
         ))
         for team, leads_to_assign_ids in leads_per_team.items():
+            total_assigned_preffered_count = 0
+            total_preferred_count = 0
```

Comment:
total and count may be redundant ^^
```suggestion
            preferred_count = 0
```